ontology/yaml/resources/HVAC/entity_types/AHU.yaml

```diff
@@ -17,6 +17,22 @@
 ########################
 
 ### Single Zone Units ###
+AHU_SS_HTWHLSTC_SFVSC_EFVSC_EDM_SDM_RFC_SFC_FDPM2X_RTM:
+  guid: "e0627d5a-99cd-4edd-824b-636428427980"
+  description: "Single zone AHU, with supply and return fans, heatwheel and fan speed flowrate control"
+  is_canonical: temperature
+  implements:
+  - AHU
+  - SS
+  - HTWHLSTC
```

Comment:
If this has a heat wheel is it actually a DOAS unit (where the supply and exhaust air streams are separated from each other)?

Reply:
@cstirdivant - thanks have changed to DOAS unit, no need for SS abstract type. Ciaran